linode/manager
# M3-5617: Update Tooltip stories

## Description
- Converts CopyTooltip to a functional component
    - Hovering over the component now displays a "Copy" tooltip
    - Remove unused `standAlone` and `ariaLabel` prop
- Converts CopyTooltip story to mdx format
- Converts HelpIcon story to mdx format
- Removed unnecessary `size` prop from HelpIcon

## How to test
- `yarn storybook`
- Check stories under http://localhost:6006/?path=/docs/ui-elements-tooltip
- Check the app for any regressions related to the refactoring
